lib/wrappers/genesisProtocol.ts

```diff
@@ -852,43 +854,65 @@ export class GenesisProtocolWrapper extends ContractWrapperBase implements Schem
 
     params = Object.assign({},
       {
-        boostedVotePeriodLimit: 60,
+        boostedVotePeriodLimit: 604800, // 1 week
```

Comment:
Does these default values are the same values used by the migration script ?
If yes, so I suggest to define that externally (e.g GenesisProtocol default params or something) and import/use both here and in the migration script

Reply:
So response below